imports/methods/TypedMethod.ts

```diff
@@ -133,10 +133,60 @@ export default class TypedMethod<
   }
 
   call<T>(...args: TypedMethodCallArgs<T, Args, Return>): void {
-    return Meteor.call(this.name, ...args);
+    try {
+      if (Bugsnag.isStarted()) {
+        Bugsnag.leaveBreadcrumb('Meteor method call', {
+          method: this.name,
+          arguments: typeof args[0] === 'object' ? EJSON.stringify(args[0]) : undefined,
+        }, 'request');
+      }
+      return Meteor.call(this.name, ...args);
+    } catch (error) {
```

Comment:
Does `Meteor.call` throw?  I thought it took a final callback argument and signaled failure via passing an error as the first arg to the callback function.  I think if you want to catch errors here you'll need to wrap the callback function the user passed, or add one if they didn't pass one.

Reply:
Yes, you're right of course. (It does, theoretically, throw on the server, where if you don't pass a callback it's invoked synchronously, but we don't do that anymore and I'm fine dropping that functionality).

Now that I'm no longer on a plane with no wifi I've actually tested this - you can see an example error reported here: https://app.bugsnag.com/death-and-mayhem/jolly-roger-dev/errors/63c8133d2264e90008beed10